packages/repeater/src/lib/Repeater.ts

```diff
@@ -1,56 +1,23 @@
-import {
-  ExecuteRequestEventHandler,
-  RegisterRepeaterCommand,
-  RegisterRepeaterResult,
-  RepeaterRegisteringError,
-  RepeaterStatusEvent
-} from '../api';
-import { RepeaterStatus } from '../models';
-import { Configuration, EventBus, Logger } from '@sectester/core';
-import { gt } from 'semver';
-import chalk from 'chalk';
-import Timer = NodeJS.Timer;
+import { RepeaterBus } from '../bus';
```

Comment:
It looks like this bus is purely a repeater. I don't understand why this separation is needed, as it doesn't seem to contribute to the code quality.

Reply:
merged